packages/css/src/Layout/Header/Client/NavBar/NavBar.scss

```diff
@@ -0,0 +1,43 @@
+@use "../../../../common/common.client.scss" as common;
+
+.af-navbar-container {
+  display: flex;
+  height: 5rem;
+  margin: 0;
+  padding: 0;
+  flex-direction: row;
+  align-items: center;
+  gap: 1.6rem;
+  list-style-type: none;
+}
+
+.af-navbar-item {
+  height: 100%;
+  cursor: pointer;
+
+  &__link {
+    display: inline-block;
+    box-sizing: border-box;
+    min-width: 8.438rem;
+    height: 100%;
+    padding: 1.75rem 1rem;
+    font-family: common.$font-family-base;
+    font-size: 1.125rem;
+    font-weight: 400;
+    line-height: 1.406rem;
```

Comment:
tu peux mettre 1.4rem, pas besoin d'une telle précision 😅

Reply:
J'ai modifié le rôle pour une meilleur accessibilité

Reply:
https://dev.to/ilizette/how-to-build-an-accessible-navigation-menu-1omk